entries/c/cardmarket.com.json

```
@@ -0,0 +1,12 @@
+{
+  "CardMarket": {
+    "domain": "cardmarket.com",
+    "recovery": "https://cardmarket.com/en/Login/LostDevice",
+    "tfa": [
+      "totp"
+    ],
+    "keywords": [
+      "gaming"
```

I think the retail category would be a better fit as, unlike Steam & Origin that are retail platforms and game launchers, CardMarket is only a retail platform.

```suggestion
      "retail"
```

---

That works for me